sphinx_autodoc_typehints.py

```diff
@@ -167,6 +167,12 @@ def process_docstring(app, what, name, obj, options, lines):
                         insert_index = i
 
                 if insert_index is not None:
+                    if insert_index == len(lines):
+                        # Ensure that :rtype: doesn't get joined with a
+                        # paragraph of text, which prevents it being
+                        # interpreted.
+                        lines.append('')
+                        insert_index += 1
```

Comment:
A couple minor code style changes: add an empty line after the inner "if" block and reduce the number of comment lines (they can be longer). Otherwise this looks good.

Reply:
Sure, I can make those changes tomorrow. What's your preferred wrap column for comments?

Reply:
100 (so wraps after 99), like the rest of the code.

Reply:
Done.